[Example 10] RRbot with GPIO interfaces (#256)
ros-controls/ros2_control_demos: example_10/README.md

> # ros2_control_demo_example_1

Reviewer: ```suggestion
# ros2_control_demo_example_10
```

>    *RRBot* - or ''Revolute-Revolute Manipulator Robot'' - a simple position controlled robot with one hardware interface. This example also demonstrates the switching between different controllers.

Author: ```suggestion
   *RRBot* - or ''Revolute-Revolute Manipulator Robot'' - with GPIO interfaces.
```
